src/profiles/gmelodie.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Gabriel Cruz
+profilepic: gmelodie.png
+github: gmelodie
+twitter: gmelodiecruz
+linkedin: gabriel-de-melo-cruz
+description: I'm a CS sophomore from University of São Paulo, Brazil currently living in France and crazy about Python and Cybersecurity
+lat: '48.389999'
+long: ''-4.490000
```

Comment:
Hey @gmelodie,
Thanks for contributing!

Just a small error in quotes.🙈

Reply:
Oh shoot! First day jitters :laughing: